extensions/default/src/DicomWebDataSource/utils/fixBulkDataURI.ts

```diff
@@ -20,16 +20,23 @@ function fixBulkDataURI(value, instance, dicomWebConfig) {
   // in case of the relative path, make it absolute. The current DICOM standard says
   // the bulkdataURI is relative to the series. However, there are situations where
   // it can be relative to the study too
-  if (!value.BulkDataURI.startsWith('http') && !value.BulkDataURI.startsWith('/')) {
-    if (dicomWebConfig.bulkDataURI?.relativeResolution === 'studies') {
-      value.BulkDataURI = `${dicomWebConfig.wadoRoot}/studies/${instance.StudyInstanceUID}/${value.BulkDataURI}`;
+  const { BulkDataURI } = value;
```

Comment:
This fixes some issues display PDF/video from external data sources (eg not default).  Could be part of another PR